src/components/MetricsPane/SelectedProjectMetrics.jsx

```diff
@@ -60,7 +61,7 @@ export const SelectedProjectMetrics = ({ selectedProject, setSelectedProject })
         </CloseButton>
       </ProjectCard>
 
-      {project_admins ? (
```

Comment:
[Using ternaries to render conditional JSX ](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)is generally a best practice. Is there an explicit reason the ternary is being removed here?

Reply:
Also can you like the original ticket in the PR description?